cypress/integration/home.spec.ts

```diff
@@ -0,0 +1,9 @@
+describe('Home screen', () => {
+  it('renders the correct heading', () => {
+    // Start from the index page
+    cy.visit('http://localhost:3000/');
+
+    cy.url().should('include', '/');
+    cy.get('h1').contains('Welcome to Next.js!');
```

Comment:
What about configuring [test-id-attribute](https://nimblehq.co/compass/development/code-conventions/javascript/cypress/#element-selector) so that we can [select](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements) element by `data-test-id`?

Reply:
Thanks, added in https://github.com/nimblehq/til-web/pull/17/commits/4d0e0c0f2e5b23d38a6fb4b2ff5d2c212afeb9c3